pkg/kubelet/cm/dra/manager.go

```diff
@@ -130,10 +123,16 @@ func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 
 				// Walk through each resourceHandle
 				for _, resourceHandle := range resourceHandles {
+					// If no KubeletPluginName is provided, we use the DriverName
+					pluginName := resourceHandle.KubeletPluginName
```

Comment:
Wouldn't empty KubeletPluginName fail a validation in pkg/apis/resource/validation/validation.go ?

Reply:
no -- it is an optional field by design

Reply:
thanks. makes sense to me.